x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1016,13 +1048,28 @@ describe('when on the endpoint list page', () => {
 
         const activityLogCallout = await renderResult.findByTestId('activityLogNoDataCallout');
         expect(activityLogCallout).not.toBeNull();
+        // scroll to the bottom by pressing down arrow key
+        // and keep it pressed
+        userEvent.keyboard('ArrowDown>');
+        // lift key up after half a second
+        // should give enough delay to move the callout
+        // past the date range filters and expose the bottom content
+        setTimeout(() => {
+          userEvent.keyboard('/ArrowDown');
+        }, 500);
+        expect(await renderResult.queryByTestId('activityLogLoadMoreTrigger')).toBeNull();
```

Comment:
Should this expect be inside the setTimeout? I'm wondering if you are waiting 500ms to release the ArrowDown key but not for the expect. Maybe you could use the [waitforelementtoberemoved](https://testing-library.com/docs/dom-testing-library/api-async/#waitforelementtoberemoved)  instead

Reply:
I used `waitFor` fc26a228798f6a09c440b571a86b6f05ae386d14